### DISCUSSION TITLE
Copilot extension for Panic Nova editor

### DISCUSSION THREAD

**Discussion by michaelaguiar** [community/community#48056] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Would LOVE to see an extension to work with Panic's Nova editor! Any plans on releasing for more editors?

**Comment by Nebula-Developer** [comment]
Nova doesn't have the popularity other editors have, but I'm sure they will plan to extend to more editors in the future, although that may take quite a while.
